_docs/aws_resources.md

```diff
@@ -8,6 +8,168 @@ permalink: /aws_resources.html
+
+As a turnkey solution, Data Pipeline works with most datasets, and can be configured through a drag-and drop interface. However, its flexibility comes at a cost, even when your import or export fails. For larger projects, such as company databases, where information needs to be consistantly moving, this is a great solution. However, for smaller projects, it can easily eat through the budget given for the Hackathon, despite it's effectiveness. As such, smaller projects are safer with building their own import/export functions with Lambda.
+
+### Lambda
+This option takes a bit of programming, and can be frustrating to implement, as it is not as flexible as Data Pipeline can be with converting information. However, it's a heckuva lot cheaper for small projects, and the user is in full control over what they are doing.
+
+Lambda is a scripting service on AWS which works based on triggers, code, and output destination. For importing/exporting data, you would need to build your own Lambda function to handle for cases. It's not flexible like pipeline, and you will need to handle the parsing logic yourself.
+
+However - this solution works great for smaller projects. Learning how to set triggers for importing data, and writing code for exporting your data, allows for the developer to have full control over what AWS is doing with their data, and allows you to avoid any hidden fees. It can serve your exact purpose, and can be changed as needed. Generally, for independant projects on a budget, making your own Lambda function would be the best choice. It's also a lot of fun!
+
+#### Import using DynamoDB
+
+.JSON
+```python
+import json
+import boto3 #sdk
+s3_client = boto3.client('s3')
+dynamodb = boto3.resource('dynamodb')
+
+# BOTO 3 DOCS: https://boto3.amazonaws.com/v1/documentation/api/latest/index.html
+# Python vers 3.6
+
+def lambda_handler(event, context):
+    print(str(event))
+    #retrieves name of S3
+    bucket = event['Records'][0]['s3']['bucket']['name']
+    #retrieves name of actual file in S3
+    json_file_name= event['Records'][0]['s3']['object']['key']
+    #retrieves the json info location in S3
+    json_object = s3_client.get_object(Bucket = bucket, Key = json_file_name)
+    #reads S3 info
+    jsonFileReader = json_object['Body'].read()
+    #parses info into 'dict' format
+    jsonDict = json.loads(jsonFileReader)
+    #grabs the table to import into
+    table = dynamodb.Table('FactsImport')
+    #goes through each item within the dict
+    #necessary for json files with more than one object
+    for k in jsonDict
+        #places each item within table
+        table.put_item(Item = k)
+```
+
+#### Export using DynamoDB
+
+.CSV (by hand)
+```
```

Comment:
Recommend doing this in rendered text instead of a code block -- it'll also look nice with bullets :) 

Also, adding subheadings to help keep the functions visually organized

```suggestion
##### Export to CSV Using Built-in Functionality

```